### PR TITLE
Add more tests for storage

### PR DESCRIPTION
**Pull request by edaena**

### CODE REVIEW

**Comment by edaena** [comment]
Coverage up from 33.3% to 67.3%

File: src/lib/azure/storage.ts [hunk 1]
```diff
@@ -149,7 +149,7 @@ export const getStorageAccountKeys = async (
     accountName
   );
 
-  if (typeof keysResponse.keys !== undefined) {
+  if (typeof keysResponse.keys !== "undefined") {
```
Comment (sarath-p) [inline src/lib/azure/storage.ts:152]: Did you try `!typeof keysResponse.keys` option?

**Review by sarath-p — APPROVED** [review]
Looks good. Added a one comment.

File: src/lib/azure/storage.ts [hunk 2]
```diff
@@ -233,7 +233,8 @@ export const getStorageAccount = async (
       resourceGroup
     );
 
-    if (accounts === undefined || accounts === null) {
+    if (typeof accounts === "undefined" || accounts === null) {
+      logger.info("__no");
```
Comment (andrebriggs) [inline src/lib/azure/storage.ts]: @edaena What does `__no` mean?

**Review by andrebriggs — APPROVED** [review]

File: src/lib/azure/storage.ts (hunk 2 above)
Comment (edaena) [reply src/lib/azure/storage.ts]: ah, I need to remove that useless log.

File: src/lib/azure/storage.ts (hunk 1 above)
Comment (edaena) [reply src/lib/azure/storage.ts:152]: sync'ed offline. typeof returns a string, therefore the comparison needs to be to "undefined" not just undefined.

**Pull request merged**